ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Freetype/tasks/main.yml

```diff
@@ -39,3 +39,44 @@
     state: present
   when: (freetype_installed.stat.exists == false)
   tags: Freetype
+
+####################################
+# Pre compiled version of freetype #
+####################################
+
+- name: Test if freetype is already installed
+  win_stat:
+    path: 'C:\openjdk\freetype'
+  register: freetype_compiled_installed
+  tags: Freetype
+
+- name: Test if freetype is already downloaded
+  win_stat:
+    path: 'C:\temp\freetype.zip'
+  register: freetype_compiled_download
+  tags: Freetype
+
+- name: Download pre compiled freetype.zip
+  win_get_url:
+    url: https://ci.adoptopenjdk.net/userContent/freetype.zip
```

Comment:
It would be good to document where this comes from originally.

Reply:
Right, but we should track where the *original* source came from for freetype, which version was used, how it was compiled, etc.  That's best done in the repo housing the zip rather than here where the zip is used.

Reply:
Freetype role being added in https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/566 superceding this